indico/modules/events/contributions/models/contributions.py

```diff
@@ -454,6 +454,12 @@ def get_non_inheriting_objects(self):
         """Get a set of child objects that do not inherit protection."""
         return get_non_inheriting_objects(self)
 
+    def is_user_associated(self, user, check_abstract=False):
+        if user is None:
+            return False
+        if check_abstract and self.abstract and self.abstract.submitter == user:
+            return True
+        return any(pl.person.user == user for pl in self.person_links if pl.person.user)
```

Comment:
FWIW, the `if pl.person.user` isn't necessary since `user` is guaranteed to be not None here. It doesn't harm though so we can keep it